boilerplate/src/configs/knexfile.js

```diff
@@ -0,0 +1,44 @@
+// Update with your config settings.
```

Comment:
Is there any way to centralise this? E.g. the database settings are repeated quite a few times, it would be good if there was a way to have a single config file to hold the hosts, usernames, etc. But if not, it's also a good thing to mention in the readme.

Reply:
This should be dealt with another task where we'll create migrations to populate database

Reply:
Alright.